src/ui/server/controllers/insurance/policy-and-export/single-contract-policy/index.ts

```diff
@@ -132,5 +147,20 @@ export const post = async (req: Request, res: Response) => {
     }
   }
 
-  return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${ROUTES.INSURANCE.POLICY_AND_EXPORTS.ABOUT_GOODS_OR_SERVICES}`);
+  try {
+    // save the application
+    const populatedData = mapSubmittedData(req.body);
+
```

Comment:
Can we add a void check for `populatedData` please.

Reply:
Hey @abhi-markan I don't think we need to, `mapSubmittedData` will always return an object?